.github/workflows/kind.yml

```diff
@@ -72,7 +72,22 @@ jobs:
     - name: Run e2e tests
       run: |
         mkdir log
-        ANTREA_LOG_DIR=$PWD/log ./ci/kind/test-e2e-kind.sh --encap-mode encap
+        mkdir test-e2e-encap-coverage
+        ANTREA_LOG_DIR=$PWD/log ANTREA_COV_DIR=$PWD/test-e2e-encap-coverage ./ci/kind/test-e2e-kind.sh --encap-mode encap --coverage
+    - name: Tar coverage files
+      run: tar -czf test-e2e-encap-coverage.tar.gz test-e2e-encap-coverage
+    - name: Upload coverage for test-e2e-encap-coverage
+      uses: actions/upload-artifact@v2
+      with:
+        name: test-e2e-encap-coverage
+        path: test-e2e-encap-coverage.tar.gz
+    - name: Codecov
+      uses: codecov/codecov-action@v1.0.13
```

Comment:
why do you specific `codecov/codecov-action@v1.0.13` instead of `codecov/codecov-action@v1`, which should use the latest release for the v1 version of the action?

Reply:
> why do you specify `codecov/codecov-action@v1.0.13` instead of `codecov/codecov-action@v1`, which should use the latest release for the v1 version of the action?

I copied it from  https://github.com/vmware-tanzu/antrea/blob/master/.github/workflows/integration.yml#L23
Maybe we all need to change to use @v1? @weiqiangt

Reply:
I didn't find where tells that `v1` will always track the latest version. If so, we should use `v1` instead of a specific version.

Reply:
From  https://github.com/codecov/codecov-action, it recommends to use @v1, so changed all codecov action to use @v1